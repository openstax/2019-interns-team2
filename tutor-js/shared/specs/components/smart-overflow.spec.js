import React from 'react';
import SmartOverflow from '../../src/components/smart-overflow';

class TestChildComponent extends React.Component {
  render() { return (
    <span>
      i am a test
    </span>
  ); }
}

describe('SmartOverflow Component', function() {
  let props = null;

  beforeEach(() =>
    props = {
      className: 'testing',
      children: <TestChildComponent />,
    }
  );

  it('renders with className', () => {
    expect.snapshot(<SmartOverflow {...props} />).toMatchSnapshot();
  });
});
