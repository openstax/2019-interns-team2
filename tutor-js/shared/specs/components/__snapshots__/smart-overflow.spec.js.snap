// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmartOverflow Component renders with className 1`] = `
<div
  className="openstax-smart-overflow testing"
>
  <span>
    i am a test
  </span>
</div>
`;
